pkg/kubelet/winstats/winstats.go

```diff
@@ -0,0 +1,127 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package winstats provides a client to get node and pod level stats on windows
+package winstats
+
+import (
+	"time"
+
+	cadvisorapi "github.com/google/cadvisor/info/v1"
+	cadvisorapiv2 "github.com/google/cadvisor/info/v2"
+)
+
+// Client is an object that is used to get stats information.
+type Client struct {
```

Comment:
The client is a struct because it is only used to keep a reference of a `winNodeStatsClient` interface which is either a `PerfCounterNodeStatsClient` or a `fakeWinNodeStatsClient`.

Reply:
Using an interface will provide an easy way for other packages to mock this.

Reply:
Done, I made `Client` a interface and made a separate `StatsClient` that implements the interface.